ecommerce/backend/API/views/shopping_cart.py

```diff
@@ -15,4 +17,36 @@ def list_shopping_cart(request, id): #userId
         return Response(status=status.HTTP_400_BAD_REQUEST)
     sc_items = ShoppingCartItem.objects.filter(customer_id=id).values('id', 'product_id', 'amount')
     sci_serializer = shopping_cart_item_serializer.ShoppingCartItemSerializer(sc_items, many=True)
-    return Response(sci_serializer.data)
\ No newline at end of file
+    return Response(sci_serializer.data)
+
+
+@api_view(['POST'])
+@permission_classes([permissions.AllowAnonymous])
```

Comment:
Can you add authorization and permission check here.
"permissions.IsCustomerUser" can be used for this request. 
And now, if I send request using token issued for user with id 5, to add item to shopping chart of user with id 6, system accepts it.
If you check token's owner at the beginning it will be secure.

You can do it this way;

if request.user.pk != id
//here you should return Http403Forbidden
else
//continue